addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -225,6 +225,9 @@
         <field name="priority">999</field>
         <field name="arch" type="xml">
             <search position="inside"/>
+            <xpath expr="//search/filter[@name='inactive']" position='attributes'>
+                <attribute name='invisible'>1</attribute>
+            </xpath>
```

Comment:
You should instead move the filter from `view_task_search_form_base` to `view_task_search_form_project_fsm_base`

Reply:
By moving the inactive filter to the `view_task_search_form_project_fsm_base` view, it will be visible in project sharing, since the `view_task_search_form_project_base` search view inherits from `view_task_search_form_project_fsm_base`. We cannot move/remove the filter in the stable version.

what do you think?

Reply:
It's the opposite, `view_task_search_form_project_fsm_base` inherits from `view_task_search_form_project_base`. But you are right, it's better to avoid doing that kind of stuff in stable.
We can do what you did for stable versions and move the field in master. 🙂